src/chi/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,196 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+  .m-dropdown {
+    display: inline-block;
+    position: relative;
+
+    .m-dropdown__menu {
+      left: 0;
+      min-width: 100%;
+      position:absolute;
+      z-index: 1;
+    }
+
+    &__trigger {
+
+      &:not([data-position^="left"]) {
+        &:after {
+          background-image: url(../html-select/icon-chevron-thin-down.svg);
+          background-position: center center;
+          background-repeat: no-repeat;
+          background-size: .75rem .75rem;
+          content: '';
+          display: block;
+          height: 1rem;
+          margin-left: 0.75rem;
+          transition: transform 0.2s ease-in-out;
+          width: 0.75rem;
+        }
+        &.-active:after {
+          transform: rotate(-180deg);
```

Comment:
Let's change to `transform: rotate(180deg);` to match global menu and ensure all other values below rotate clockwise as well